R/databricks_connection.R

```diff
@@ -77,11 +77,6 @@ new_databricks_connection <- function(scon, guid) {
   session <- invoke_static(sc, r_driver_local, "getDriver", guid) %>%
     invoke("sqlContext") %>%
     invoke("sparkSession")
-  # Apply user-specified fs.azure.* config values
-  fs.azure.prefix <- "fs.azure."
```

Comment:
Truth be told, I don't think this has ever worked... Thanks for finding & deleting it!